BackendTaskPilot/Program.cs

```diff
@@ -0,0 +1,38 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+ExchangeRateService service = new ExchangeRateService();
```

Comment:
It's a good practise not to use the concrete implementation. Instead use dependency injection to ensure decoupling, better flexibility, and testability. The DI container will also take care of the object's life cycle for us. A few steps could be:

- Extract the class **ExchangeRateService** from the **Program.cs**
- Create an interface called **IExchangeRateService** 
- Register the dependency injection. For more: https://learn.microsoft.com/en-us/dotnet/core/extensions/dependency-injection